### PR TITLE
Update Set-WindowsSearchSetting.md

### PR DESCRIPTION
**Pull request by RAJU2529**
added a new command with a description 
**Set-WindowsSearchSetting -EnableMeteredWebResultsSetting $True**

article link
**https://learn.microsoft.com/en-us/powershell/module/windowssearch/set-windowssearchsetting?view=windowsserver2022-ps**

Screenshot 2023-10-21 130756

### CODE REVIEW

File: docset/winserver2022-ps/windowssearch/Set-WindowsSearchSetting.md [hunk 1]
```diff
@@ -33,17 +33,24 @@ You can also specify SafeSearch settings.
...
 ```
 PS C:\> Set-WindowsSearchSetting -EnableWebResultsSetting $True -SafeSearchSetting "Strict" -SearchExperienceSetting "PersonalizedAndLocation"
 ```
 
 This command modifies Windows Search settings.
-The command enables the use of search history and specific location of the user by specifying a value of PersonalizedAndLocation for the *SearchExperienceSetting* parameter.
+The command enables the use of search history and the specific location of the user by specifying a value of PersonalizedAndLocation for the *SearchExperienceSetting* parameter.
 The command enables web results and sets SafeSearch to a value of Strict.
 
+### Example 3: Enable Metered Web Search
```
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/windowssearch/Set-WindowsSearchSetting.md:47]:
```suggestion
### Example 3: Enable metered web search
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

File: docset/winserver2022-ps/windowssearch/Set-WindowsSearchSetting.md [hunk 2]
```diff
@@ -33,17 +33,24 @@ You can also specify SafeSearch settings.
...
-The command enables the use of search history and specific location of the user by specifying a value of PersonalizedAndLocation for the *SearchExperienceSetting* parameter.
+The command enables the use of search history and the specific location of the user by specifying a value of PersonalizedAndLocation for the *SearchExperienceSetting* parameter.
 The command enables web results and sets SafeSearch to a value of Strict.
 
+### Example 3: Enable metered web search
+```
+PS C:\> Set-WindowsSearchSetting -EnableMeteredWebResultsSetting $True
+```
+
+This command enables metered web searching results only.
```
Comment (get-itips) [inline docset/winserver2022-ps/windowssearch/Set-WindowsSearchSetting.md:52]:
```suggestion
This command enables Windows Search to displays web results and suggestions while using metered connections.
```

**Review by get-itips — APPROVED** [review]

File: docset/winserver2022-ps/windowssearch/Set-WindowsSearchSetting.md [hunk 3]
```diff
@@ -33,17 +33,24 @@ You can also specify SafeSearch settings.
...
-The command enables the use of search history and specific location of the user by specifying a value of PersonalizedAndLocation for the *SearchExperienceSetting* parameter.
+The command enables the use of search history and the specific location of the user by specifying a value of PersonalizedAndLocation for the *SearchExperienceSetting* parameter.
 The command enables web results and sets SafeSearch to a value of Strict.
 
+### Example 3: Enable metered web search
+```
+PS C:\> Set-WindowsSearchSetting -EnableMeteredWebResultsSetting $True
+```
+
+This command enables Windows Search to displays web results and suggestions while using metered connections.
```
Comment (JohanFreelancer9) [inline docset/winserver2022-ps/windowssearch/Set-WindowsSearchSetting.md:52]:
```suggestion
This command enables Windows Search to display web results and suggestions while using metered connections.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @RAJU2529, minor correction, please.

**Review by msbemba — APPROVED** [review]

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@tiburd 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**